ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/validation/ShaclShapeValidator.java

```diff
@@ -14,14 +15,9 @@ protected void initializeShapes() {
 		shapes = Shapes.parse(FILE_NAME);
 	}
 
-	@Override
-	public boolean supports(Class<?> clazz) {
-		return Model.class.isAssignableFrom(clazz);
-	}
-
-	@Override
-	public void validate(Object target, Errors errors) {
```

Comment:
can we extract this implementation to the abstract parent class? Same for the other similar impls